static/app/views/settings/account/notifications/notificationSettingsByProjects.tsx

```diff
@@ -74,6 +90,13 @@ class NotificationSettingsByProjects extends AsyncComponent<Props, State> {
     );
   };
 
+  handleOrgChange = (option: {label: string; value: string}) => {
+    // handleOrgChange(option: {label: string; value: string}) {
+    const self = this;
+    this.props.handleOrgChange(option);
+    setTimeout(() => self.reloadData(), 0);
```

Comment:
```suggestion
    this.props.handleOrgChange(option);
    setTimeout(() => this.reloadData(), 0);
```

You don't need a `this` alias as arrow functions inherit scope binding from the defining scope.